gravityflow/gravityflowblocks
# Fix script enqueue for WordPress 5.8.1 and up

## Description

In WordPress 5.8.1 and later depending on `wp-editor` and having the depending script force them all into the head of admin will break the editor. This makes our block script load in footer and patches that.

This pr bumps version to 0.4 as well

## Testing instructions
* Use the WordPress beta testing plugin and put yourself on the latest 5.9 or 5.8.1
* Activate this addon
* Note that the editor loads and our block works correctly.
